README.md

```diff
@@ -38,7 +38,13 @@ $ git hist
 
 ![Screenshot](http://i.imgur.com/D5AqNUz.png)
 
-*If your terminal displays weird characters, please make sure you're running the latest version of `less` before reporting an issue. Thanks! :beers:*
+*If your terminal displays weird characters, please make sure you're running at least version `471` of `less` before reporting an issue. Thanks! :beers:*
+
+*You can install `less` via [Homebrew](http://brew.sh):*
```

Comment:
I thought about it too, but then I realized less-471 is in _beta_ since 14 Dec 2014. I don’t know, I guess it’s okay to mention it.